AutomationHelpers/Ranorex.AutomationHelpers.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>Ranorex.AutomationHelpers</id>
-        <version>1.5.1</version>
+        <version>1.5.2</version>
```

Comment:
No need to update the Helpers version, Eyes is delivered separately (separate Nuget package)